context.playrandomvideo/addon.xml

```diff
@@ -12,8 +12,9 @@
 		</item>
 	</extension>
 	<extension point="kodi.addon.metadata">
-		<summary lang="en">Plays a random video from the selected item.</summary>
-		<description lang="en">Plays a random video from the selected item. It can play random episodes from TV shows, movies from genres/sets/years/tags, and videos from playlists, the filesystem, and just about anything else, other than plugins.</description>
+		<broken lang="en">Context menu has been merged into the script, a separate add-on is no longer needed.</broken>
```

Comment:
You shouldn't use the broken tag for that info. This will cause the addon to be uninstalled from all systems.